website/docs/quick-start/installation/docker-compose.mdx

```diff
@@ -48,12 +48,14 @@ services:
     image: registry.tabbyml.com/tabbyml/tabby
     entrypoint: /opt/tabby/bin/tabby-cpu
     command: serve --model StarCoder-1B --chat-model Qwen2-1.5B-Instruct
+    environment:
+     - LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/compat:$LD_LIBRARY_PATH #Workaround to run in cpu mode, see see https://github.com/TabbyML/tabby/issues/2634#issuecomment-2244530283
```

Comment:
This workaround will not work on a system without a CUDA installation, thus it is largely not very useful. I recommend using the manylinux2014 binary build for CPU-only deployment.

Reply:
That's the weird thing: I do not have any cuda installation on that system. Not setting this variable still gives me tons of error messages that cuda was not found even though I am using the tabby-cpu entrypoint. Is this entrypoint using the mentioned manylinux-2014 binary or is this something else and not available in the docker image?

Reply:
the docker image bundled the llama-cpp-server binary with cuda dependencies, thus shall not runnable for a cpu only environment 🤔

Reply:
Well, it is. Or I am missing something that this virtual server can do that I do not know

Reply:
Sure, I recommend removing this part from the PR so that the remaining parts can be merged (contributing to the website + deploying behind a proxy).

Reply:
Commits dropped, please review